convert2rhel/unit_tests/actions/report_test.py

```diff
@@ -175,18 +206,63 @@ def test_summary(results, expected_results, include_all_reports, caplog):
             },
             True,
             [
-                (0, "(ERROR) ErrorAction.ERROR: ERROR MESSAGE"),
-                (1, "(OVERRIDABLE) OverridableAction.OVERRIDABLE: OVERRIDABLE MESSAGE"),
-                (2, "(SKIP) SkipAction.SKIP: SKIP MESSAGE"),
-                (3, "(WARNING) WarningAction.WARNING: WARNING MESSAGE"),
-                (4, "(SUCCESS) PreSubscription: All good!"),
+                r"\(ERROR\) ErrorAction.ERROR: ERROR MESSAGE",
+                r"\(OVERRIDABLE\) OverridableAction.OVERRIDABLE: OVERRIDABLE MESSAGE",
+                r"\(SKIP\) SkipAction.SKIP: SKIP MESSAGE",
+                r"\(WARNING\) WarningAction.WARNING: WARNING MESSAGE",
+                r"\(SUCCESS\) PreSubscription: All good!",
             ],
         ),
     ),
 )
 def test_summary_ordering(results, include_all_reports, expected_results, caplog):
-    report.summary(results, include_all_reports)
 
-    # Get the order and the message
-    log_ordering = ((index, record.message) for index, record in enumerate(caplog.records))
-    assert expected_results == list(log_ordering)
+    report.summary(results, False, include_all_reports)
+
+    pattern = []
+    for entry in expected_results:
+        pattern.append(entry)
+
+    # Prove that all the messages occurred and in the right order.
+    message = caplog.records[-1].message
+    pattern = ".*".join(pattern)
+
+    assert re.search(pattern, message, re.DOTALL)
```

Comment:
You have two changes regarding ordering, which one we would like to accept? This one or the one in https://github.com/oamg/convert2rhel/pull/798

Reply:
My plan is to merge the changes together once one of the PRs is merged.

Reply:
Okay, #798 has been merged.  I have rebased this and reconciled the unittests.